### PR TITLE
NAS-109476 / 21.08 / Explain replicate_tooltip to use new name_regex option

### PR DESCRIPTION
**Pull request by themylogin**
@Mrt134 we will introduce a new field in another PR https://jira.ixsystems.com/browse/NAS-110543

With this PR I want to change existing field's tooltip to refer to that newly added field. Please review their names and descriptions in the JIRA issue and here.

### CODE REVIEW

**Review by Mrt134 — APPROVED** [review]
Text looks good!

**Comment by undsoft** [comment]
Probably want to wait before merging until we actually have the new field.

**Comment by RehanY147** [comment]
@undsoft Is this ready to be merged yet?

**Comment by undsoft** [comment]
No, blocked until this is implemented: https://jira.ixsystems.com/browse/NAS-110543

File: src/app/helptext/data-protection/replication/replication.ts [hunk 1]
```diff
@@ -108,8 +108,9 @@ export default {
 
   replicate_placeholder: T('Full Filesystem Replication'),
   replicate_tooltip: T('Completely replicate the selected dataset. The target \
-dataset will have all of the properties, snapshots, child datasets, and clones \
-from the source dataset.'),
+dataset will have all of the source dataset\'s properties, child datasets, \
+clones and snapshots that match the specified naming schema. Set \
+Snapshot Name Regular Expression to .* to replicate all snapshots.'),
```
Comment (undsoft) [inline src/app/helptext/data-protection/replication/replication.ts:113]: @themylogin So if I don't read the tooltip and don't set anything in either Regular Expression or naming schema, what would happen?

**Comment by undsoft** [comment]
@RehanY147 Can this now be merged?

**Comment by RehanY147** [comment]
Looks like it. The linked ticket has been marked `done`.

**Pull request merged**